non_www/tabs/school/download_national.html

```diff
@@ -0,0 +1,75 @@
+<div class="container">
+  <div class="row maintext" id="maintext">
+    <div class="col-md-2 text-left" style="padding-top:10px;">
+      <div style="padding-bottom:10px;"><a class="natdownloadlinks" href="#natschoolzones">Zones data</a></div>
+      <div style="padding-bottom:10px;"><a class="natdownloadlinks" href="#natschooldestinations">Schools data</a></div>
+    </div>
+
+    <div class="col-md-9">
+      <div class="col-md-12 text-left" style="padding-top:10px;">
+        <h3>National Data  - school layer</h3>
+        <p>This page contains data downloads for the schools layer for the whole of England. These are available as CSV files containing baseline and scenario attribute data for zones and schools, and as geographic files that can be read by R (.Rds) or GIS programs such as QGIS (.geojson).</p>
+        <p>Please be aware that equivalent files for different purposes and geographies have <b>the same file name</b> when downloaded (e.g. ‘z_all.geojson’ for national zones files for both commuting and school travel).  To avoid confusion you may want to rename those files or save them in labelled folders. Note also that when <b>downloading 'csv' files using a Firefox browser</b>, the file will open onscreen in a new webpage.  If you then type "Ctrl + S" the onscreen data will download as a csv file.</p>
+      </div>
+
+      <div class="col-md-12 text-left" style="padding-top:10px;">
+        <h4><a name="natschoolzones"></a>School travel data for zones</h4>
+        <p>Download anonymised National School Census 2011 baseline data and scenario attribute data for all zones in England. These files present aggregate data for all children <b>living in a given zone</b>. </p>
+        <ul>
+          <li>Zones (local authorities)*:
+            <a href="https://github.com/npct/pct-outputs-national/raw/master/school//lad_attributes.csv" target="_blank" download="lad_attributes.csv" onclick="ga('send', 'event', 'download', 'lad_attributes.csv');">
```

Comment:
It's not all publicly accessible data. Maybe we should mention that.

Reply:
It isn't just that, https://github.com/npct/pct-outputs-national doesn't have this data in so the link will never work (or do you do something that I don't understand?)

Reply:
it is not put online yet because a) I don't want DfT to notice we have done all this work in advance of them agreeing to pay us and b) I think we probably do need to check with the Department for Education that we are allowed to put these data up. I hope we would be, since they are all aggregated to the level of area/school/route network, i.e. no OD data. So ultimately it should be the case that the data does exist uploaded on to git at the other end of the link.  obviously if we can't ultimately upload that data then we will modify this file

Reply:
Ok, might make sense, thanks for the explanation.  It might make sense to use a link like `www.example.com/lad_attributes.csv` but also fine as it is.